src/containers/proposalSnapshot/index.tsx

```diff
@@ -102,7 +102,9 @@ const ProposalSnapshot: React.FC<Props> = ({
 
   if (proposalCountIsFetched && (proposalCount === 0 || proposalCountError)) {
     return (
+      // needs ODS adjustment & release long term
```

Comment:
Let's link the Jira ticket here and on the other comment as well 🙏🏻 
```suggestion
      // TODO: remove this when fixed on ODS (APP-2994)
```